Automattic/pocket-casts-android
# Fix Galaxy Watch skip issues

## Description

This change relates to the Galaxy Watch issue many users are experiencing. 
https://github.com/Automattic/pocket-casts-android/issues/2708

The code adds some extra logging to help debug and includes some extra Media Session methods that might help.

## Testing Instructions

1. Start playing an episode on your phone
2. On your Galaxy Watch open the media controls 
3. Tap skip forwards and backwards
4. ✅ Verify the audio skips 
5. ✅ Verify logs are appearing logcat with the filter "Event from Media Session to KeyEvent"

## Checklist
